mapping.py

```diff
@@ -0,0 +1,38 @@
+# Script to map Sample IDs and obtain family ids + relationship
```

Comment:
I have no clue why the script seems so shabby. Too many variables are being assigned and too many subsets of dataframes are made. Any Ideas?

Reply:
1. Use the names in pd.read_csv to correctly name "SFARI ID" and "Repository Id". When creating "Family ID" and "Relationship", use proper names without spaces. Then there is no need to rename columns later on. That eliminates lines 10 and 11.
2. Are you using df_grouped somewhere? I don't see it.